### ISSUE TITLE
Cannot enable TLS on windows.

### ISSUE THREAD

**Issue by jackedak** [webui-dev/zig-webui#74]
On my windows machine, whenever i try to compile with TLS enabled, zig says that it cannot find SSL or crypto, and that it has searched no paths. I have OpenSSL installed, an it is in path, and i have even tried directly linking libssl.

**Comment by jackedak** [comment]
The pre-built files have the file webui-2.dll, while the program says it requires webui.dll, any idea why?

**Comment by jackedak** [comment]
It also complains if I just rename the file.

**Comment by jackedak** [comment]
The program that utilizes zig-webui, which is currently just `const std = @import("std");
const os = std.os;
const webui = @import("webui");

const main = @embedFile("main.html");

pub fn main() !void {
    var nwin = webui.newWindow();
    _ = nwin.show(main);
    webui.wait();
}
`

**Comment by jackedak** [comment]
In case it matters, the build.zig is 
```
const std = @import("std");

pub fn build(b: *std.Build) void {
    const target = b.standardTargetOptions(.{});
    const optimize = b.standardOptimizeOption(.{});
    const exe = b.addExecutable(.{
        .name = "main",
        .root_source_file = b.path("src/main.zig"),
        .target = b.host,
    });
    const zig_webui = b.dependency("zig-webui", .{
        .target = target,
        .optimize = optimize,
        .is_static = false,
    });
    exe.subsystem = .Windows;
    exe.root_module.addImport("webui", zig_webui.module("webui"));
    b.installArtifact(exe);
}
```

**Comment by jinzhongjia** [comment]
Yes, the problem does exist on Windows, but the root cause of the problem is in the upstream webui.
The build of webui cannot correctly find system libraries on windows
I won't be able to deal with this issue for a short time, but maybe I can let him take a look. @mochalins

**Comment by mochalins** [comment]
I'll take a look, thanks for the ping.

**Comment by jinzhongjia** [comment]
Past CI tests did not have tls testing. At least in the 0.11 period, tls was running normally.
After the issue is resolved, you need to add the CI test of tls

**Comment by jinzhongjia** [comment]
After about two weeks, I have a lot of free time to deal with the deposition problem of this library.

**Comment by jinzhongjia** [comment]
OK,I'm back.
I will try to fix it tomorrow

**Comment by jinzhongjia** [comment]
Well, I tried for a few hours and still didn't find the perfect solution. The zig compiler never found the ssl library correctly, even if the path was specified.
I will make additional attempts when I have time

**Comment by jinzhongjia** [comment]
Good idea, I will initiate a PR to the upstream and let webui's build.zig expose the option to set the location of the library

**Comment by jinzhongjia** [comment]
A very strange problem, the Zig's construction system cannot find a package in the LIB and INCLUDE directory I specified, but GCC can

**Comment by jinzhongjia** [comment]
My previous attempt is to change directly in the upstream, but I still can't find the library.
